style.css

```
@@ -1728,3 +1753,60 @@ kbd {
+/* Error notification banner */
+.flock-banner {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  z-index: 20000;
+  display: flex;
+  align-items: center;
+  gap: 12px;
+  padding: 12px 16px;
+  font-family: "Asap", sans-serif;
```

Stylelint (17.11.1)

[error] Expected no quotes around "Asap" (font-family-name-quotes)

(font-family-name-quotes)